docs/trajectory_gradspec.rst

```diff
@@ -0,0 +1,29 @@
+=========================================
+ The trajectory binary file specification
+=========================================
+
+The k-space trajectories are transformed to a binary file which is processed by scanner.
```

Comment:
The `.bin` file can directly be used by a scanner ? If yes which one ? If No, specifies  in doc the steps that the user would have to do to reach that goal.

Reply:
We cant give too much details without breaking stuff with Siemens. I have added minor extra info.